Plugins/Flow.Launcher.Plugin.Explorer/Search/WindowsIndex/WindowsIndex.cs

```diff
@@ -47,21 +47,22 @@ private static async IAsyncEnumerable<SearchResult> ExecuteWindowsIndexSearchAsy
             while (await dataReader.ReadAsync(token))
             {
                 token.ThrowIfCancellationRequested();
-                if (dataReader.GetValue(0) == DBNull.Value || dataReader.GetValue(1) == DBNull.Value)
+                if (dataReader.GetValue(0) is DBNull
+                    || dataReader.GetValue(1) is not string rawFragmentPath
+                    || string.Equals(rawFragmentPath, "file:", StringComparison.OrdinalIgnoreCase)
+                    || dataReader.GetValue(2) is not string extension)
```

Comment:
When will this not be a string?

Reply:
Most likely it will never be anything non-string, but we don't need anything else anyway. Code below used GetString(1) and GetString(2), so it will most likely crash if we get smth else(I believe GetString() differs from GetValue() only by hard-casting GetValue() to string). So I just filtered out this hypothetical case.

Reply:
To clarify: I never saw anything non-string there.

Reply:
Yeah, very good change 💯 , thanks for adding the additional checks.